stream-chat-android-ui-components/src/main/res/drawable-night/stream_ui_shape_search_view_background.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Does this shape need duplication, would the same background not work for the other variant?

Reply:
Tested... yes, no need to duplicate that.